pipelines/bazel-release.yml

```
@@ -509,8 +509,6 @@ steps:
       cd scripts/release
       python3 relnotes.py
 
-  - wait
```

instead of not waiting, let's just make this continue on failure (https://buildkite.com/docs/pipelines/configure/step-types/wait-step#continuing-on-failure).

---

Done.